run_node.sh

```diff
@@ -126,11 +126,20 @@ then
     fi
 
     PORT_MAP="${PORT}:${PORT}"
+    METRICS_PORT_MAP="127.0.0.1:9615:9615"
```

Comment:
Maybe we should add this port to env files? For example under `METRICS_PORT`

Reply:
Fair point!